geode-core/src/main/java/org/apache/geode/internal/cache/backup/InProgressBackupFiles.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache.backup;
+
+import static org.apache.geode.internal.cache.backup.BackupManager.DATA_STORES_TEMPORARY_DIRECTORY;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.Logger;
+
+import org.apache.geode.cache.DiskStore;
+import org.apache.geode.internal.cache.DirectoryHolder;
+import org.apache.geode.internal.cache.Oplog;
+import org.apache.geode.internal.logging.LogService;
+
+/**
+ * Creates and keeps track of the temporary locations used during a backup. Most temporary files are
+ * stored in a shared temporary directory, except for those for DiskStores. For each
+ * {@link DiskStore}, a temporary directory is created per disk directory within that disk directory
+ * (for enabling creation of hard-links for backing up the files of an {@link Oplog}).
+ *
+ * @since Geode 1.4
```

Comment:
This since is wrong. You do not need to add a since tag for internal classes

Reply:
removed